src/qt/walletframe.cpp

```diff
@@ -138,6 +138,13 @@ void WalletFrame::gotoSendCoinsPage(QString addr)
         i.value()->gotoSendCoinsPage(addr);
 }
 
+void WalletFrame::gotoManageNamesPage()
+{
+    QMap<QString, WalletView*>::const_iterator i;
+    for (i = mapWalletViews.constBegin(); i != mapWalletViews.constEnd(); ++i)
```

Comment:
Can this be a for-each loop (not sure since Qt)?  If not, please at least declare `i` in the loop instead of outside, and possibly as `auto`.